File: API/VelsikAPI/Database/Models/Response.py

class Response:
    def __init__(self, response_id, apv_question_id, user_id, answer, comment):
        self.response_id = response_id
        self.apv_question_id = apv_question_id
        self.user_id = user_id
        self.answer = answer
        self.comment = comment

    def __str__(self):
        return f"Response(reponse_id={self.response_id}, apv_question_id={self.apv_question_id}, " \
               f"user_id={self.user_id}, answer={self.answer}, " \
               f"comment={self.comment}"

File: App/velsik/lib/models/response.dart
class Response {
  int? responseId;
  int? questionId;
  int? userId;
  bool? answer;
  String? comment;
  String? questionTitle;
  String? questionText;

  Response(
    this.responseId,
    this.questionId,
    this.userId,
    this.answer,
    this.comment,
  );

  // Convert Question object to a JSON-encodable map
  Map<String, dynamic> toJson() {
    return {
      'responseId': responseId,
      'questionId': questionId,
      'userId': userId,
      'answer': answer,
      'comment': comment,
    };
  }
  
}
